tools/mapmerge2/dmm.py

```diff
@@ -84,7 +84,7 @@ def reassign_bad_keys(self, bad_keys):
             self.grid[k] = bad_keys.get(v, v)
 
     def remove_unused_keys(self, modified_keys = None):
-        unused_keys = list(set(modified_keys)) if modified_keys is not None else self.dictionary.keys()
+        unused_keys = list(set(modified_keys)) if modified_keys is not None else list(self.dictionary.keys())
```

Comment:
Does this need to be a list in the first place? Why not just use a set in both cases for more efficient removal?

Reply:
That should be fine, fixed.